keepassxc-browser/background/keepass.js

```diff
@@ -10,18 +10,13 @@ keepass.isDatabaseClosed = true;
 keepass.isKeePassXCAvailable = false;
 keepass.isEncryptionKeyUnrecognized = false;
 keepass.currentKeePassXC = '';
-keepass.requiredKeePassXC = '2.3.0';
-keepass.nativeHostName = 'org.keepassxc.keepassxc_browser';
-keepass.nativePort = null;
-keepass.keySize = 24;
+keepass.requiredKeePassXC = '2.3.1';
```

Comment:
Specific reason for the bump?

Reply:
Ubuntu Bionic has 2.3.1. I have no information if any distro has older than that anymore.